src/lib/dialog/dialog-config.ts

```diff
@@ -57,6 +57,26 @@ export class MatDialogConfig {
   /** Height of the dialog. */
   height?: string = '';
 
+  /** Min-width of the dialog. */
+  minWidth?: string = '';
+
+  /**
+   * Min-height of the dialog.
+   * For this value to be effectively applied, `height` may also need to be defined.
+   * See https://www.w3.org/TR/CSS2/visudet.html#min-max-widths
```

Comment:
I don't think that this needs to link to the CSS spec. Something along the lines of `/** min-height of the dialog */` should suffice. Same goes for the `maxHeight`.

Reply:
Done.